### ISSUE TITLE
Whitelist "ubuntu-toolchain-r-test" for Bionic

### ISSUE THREAD

**Issue by offa** [travis-ci/apt-source-safelist#410]
The _Ubuntu Toolchain Test_ PPA which provides current GCCs is blocked on _Bionic_:

```
Adding APT Sources

Disallowing sources: ubuntu-toolchain-r-test

To add unlisted APT sources, follow instructions in https://docs.travis-ci.com/user/installing-dependencies#Installing-Packages-with-the-APT-Addon
```

[Full example build log](https://travis-ci.org/offa/ci-playground/builds/579887954)

### Build system information

```
Build language: generic
Build group: stable
Build dist: bionic
```

### Travis.yml example:

```yml
language: generic

dist: bionic

matrix:
    include:
    - env: CXX=g++-9 CC=gcc-9
      addons:
        apt:
          packages:
            - g++-9
          sources: &sources
            - ubuntu-toolchain-r-test

script:
  - ${CC} --version
  - ${CXX} --version
  - cmake --version
```

**Comment by EwoutH** [comment]
@offa Did you by any chance find a workaround for this issue?

**Comment by offa** [comment]
My current workaround is using `sourceline`:
```yml
addons:
  apt:
    sources:
      - sourceline: 'ppa:ubuntu-toolchain-r/test'
```

**Comment by offa** [comment]
I'll close here, since it's no longer relevant.
